### PR TITLE
[*] PDF : tel and fax translation

### PR DESCRIPTION
**Pull request by cedricfontaine**

### CODE REVIEW

File: pdf/footer.tpl [hunk 1]
```diff
@@ -34,11 +34,11 @@
 			{if !empty($shop_phone) OR !empty($shop_fax)}
 				{l s='For more assistance, contact Support:' pdf='true'}<br />
 				{if !empty($shop_phone)}
-					Tel: {$shop_phone|escape:'html':'UTF-8'}
+					{l s='Tel:' pdf='true'} {$shop_phone|escape:'html':'UTF-8'}
```
Comment (jeromenadaud) [inline pdf/footer.tpl]: Hi,

Thank you for your contribution, what about this : ```{l s='Tel: %s' sprintf=[$shop_phone|escape:'html':'UTF-8'] pdf='true'}``` ? Like this you can preserve RTL translations.

Best regards.

**Comment by cedricfontaine** [comment]
Hello,
Yes, you're right.

**Comment by jeromenadaud** [comment]
Hi,

Tell if you will change your PR, if not I'll merge it and make a new PR for RTL code.

Best regards.

**Comment by cedricfontaine** [comment]
Ok updated. Thanks.

**Comment by jeromenadaud** [comment]
Hi,

Thanks to you @cedricfontaine,
Thank you for your contribution.

Best regards.

**Pull request merged**